tee-worker/scripts/litentry/release/deploy.sh

```diff
@@ -189,14 +213,24 @@ function restart_parachain() {
   local service_name="relay-alice"
   local description="Alice Node for Relay Chain"
   local working_directory="$TMPDIR"
-  local log_file=/tmp/parachain_dev/relay.alice.log
+  if [ "$ROOT" = true ]; then 
+    local log_file=/opt/parachain_dev/relay.alice.log
+  else 
+    local log_file=/tmp/parachain_dev/relay.alice.log
+  fi 
   local command="$POLKADOT_BIN --chain $ROCOCO_CHAINSPEC --alice --tmp --port ${AlicePort:-30336} --ws-port ${AliceWSPort:-9946} --rpc-port ${AliceRPCPort:-9936}"
 
   generate_service_file "${service_name}" "${description}" "${command}" "${working_directory}" "${log_file}"
-  cp ./${service_name}.service ~/.config/systemd/user/
+  if [ "$ROOT" = true ]; then
+    cp ./${service_name}.service /etc/systemd/system/ 
```

Comment:
Did you test it? 

I believe `/etc/systemd/system/` needs the root, I don't see you using `sudo` here

Reply:
I assumed when we call the script with `root` user, the caller will have permissions to do this, Let me know what you think.

Reply:
I'm also inclined towards having a single branch which is called by `root` ,  That would mean if anyone wants to deploy on servers, they should always have `root` permission. I like this, but my worry is that then we should give `root` permission to anyone who wants to deploy, if that's fine, then I guess we could have single branch.

Reply:
Single branch for root use only now